addons/web/static/src/views/fields/properties/property_definition.xml

```diff
@@ -15,6 +15,18 @@
                         <i class="oi oi-chevron-down ms-2"/>
                     </button>
                 </div>
+                <div t-else="">
+                    <style>
+                        .o_property_field_popover {
```

Comment:
I'm pretty sure the XML fields templates are part of the same assets bundle as the CSS and JS files actually.
Meaning that this template will always be synchronized with your SCSS counterpart, meaning this overhead is not necessary and we can put the change directly in the scss.

This kind of trick would be necessary if we modified a XML view for example, but not here.

But @std-odoo usually knows more about this stuff than me and he validated so now I have a doubt 😄

Reply:
@awa-odoo ah, your are right, it's not a view...